website/api/zkvm/composition.md

```diff
@@ -42,15 +42,15 @@ This is accomplished via `resolve`, which is called automatically when users cal
 For a more detailed dive into proof composition and RISC Zero's approach to recursive proving, check out the [Study Club recording].
 
 [`add_assumption()`]: https://github.com/risc0/risc0/blob/v1.0.1/examples/composition/src/main.rs#L29
-[assumption]: https://docs.rs/risc0-zkvm/latest/risc0_zkvm/struct.Assumption.html
-[assumptions]: https://docs.rs/risc0-zkvm/latest/risc0_zkvm/struct.Assumption.html
+[assumption]: /terminology#assumption
+[assumptions]: /terminology#assumption
```

Comment:
This could be collapsed into a single reference link.

Reply:
I'm struggling to remember/guess/google the appropriate syntax...

Reply:
`[link-id][link-text]`

Reply:
https://riptutorial.com/markdown/example/2215/reference-link

Reply:
https://github.com/risc0/risc0/pull/1960/commits/33d26bc0dc920af1264e6ea6101dd9033ce00b5f

Reply:
> [link-id][link-text]
ahh yes ok. 

> https://github.com/risc0/risc0/commit/33d26bc0dc920af1264e6ea6101dd9033ce00b5f
Thanks Nuke!